testing/python/collect.py

```diff
@@ -104,8 +104,24 @@ def test_show_traceback_import_error(self, testdir, verbose):
             else:
                 assert name not in stdout
 
+    def test_show_traceback_import_error_unicode(self, testdir):
+        """Check test modules collected which raise ImportError with unicode messages
+        are handled properly (#2336).
+        """
+        testdir.makepyfile(u"""
+            # -*- coding: utf-8 -*-
+            raise ImportError(u'Something bad happened ☺')
+        """)
+        result = testdir.runpytest()
+        result.stdout.fnmatch_lines([
+            "ImportError while importing test module*",
+            "Traceback:",
+            "*raise ImportError*Something bad happened*",
+        ])
+        assert result.ret == 2
+
 
-class TestClass(object):
+class TestClass:
```

Comment:
This seems wrong, since there was a change to `features` making everything inherit from `object` (and it's the added test above which comes from `master`)